src/server/game/Entities/Totem/Totem.cpp

```diff
@@ -132,7 +146,7 @@ void Totem::UnSummon(uint32 msTime)
 
         // Remove Sentry Totem Aura
         if (GetEntry() == SENTRY_TOTEM_ENTRY)
-            owner->RemoveAurasDueToSpell(SENTRY_TOTEM_SPELLID);
+            owner->RemoveAurasDueToSpell((uint32)TotemSpellIds::SentryTotemSpell);
```

Comment:
preferably a static_cast otherwise it looks good

Reply:
Done, ty